Jenkinsfile

```diff
@@ -73,6 +73,36 @@ pipeline {
               onlyIfSuccessful: true
           }
         }
+        /**
```

Comment:
if we want to speed up the build we would move it to the parallel block of stages

Reply:
it takes 21m 22s Wow!!!

Reply:
Oops, it ran the tests 🤦‍♂ 
Fixing..

Reply:
I can't find a way to skip phases and run the install plugin directly through `mvn install:insall` based on the unstashed state from the build phase (artifacts created through `mvn package`). I am not sure there is an option to do this in a multi-module maven project at all. 
So I will just do the additional `install` phase in the build stage and create the list and publish from there.
@kuisathaverat you see any good reason why I shouldn't do so? Shouldn't add a lot to the existing stage.

Reply:
Testing, let's see how it looks like

Reply:
Added less than one minute to the Build stage. I think this is a good solution.